src/views/ScenarioManager/ScenarioManager.js

```diff
@@ -142,6 +150,10 @@ const ScenarioManager = (props) => {
     (scenario) => hasUserPermissionOnScenario(ACL_PERMISSIONS.SCENARIO.WRITE, scenario),
     [hasUserPermissionOnScenario]
   );
+  const canUpdateScenario = useCallback(
+    (scenario) => hasUserPermissionOnScenario(ACL_PERMISSIONS.SCENARIO.WRITE, scenario),
+    [hasUserPermissionOnScenario]
+  );
```

Comment:
As mentioned in the other PR, we could try to merge this function with `canUserRenameScenario` (both functions have exactly the same code). If we want to make as few breaking changes as possible, we could remove `canUserRenameScenario` and provide `canUpdateScenario` in both props `canUserRenameScenario` and `canUpdateScenario`.